run_in_parallel.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: UTF-8 -*-
+
+"""
+based on from github "https://gist.github.com/s1ider/f13c2f163282dbec7a61"
+Customized for parallel scenarios
+Authors: i4s-pserrano
+"""
+
+import argparse
+import json
+import logging
+import os
+import sys
+import time
+from datetime import datetime
+from distutils.util import strtobool
+from multiprocessing import Process, Queue
+from subprocess import Popen, PIPE, check_output, CalledProcessError, TimeoutExpired
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+DEFAULT_COMMAND_LINE_ARGS = '--stop --no-summary'
+DEFAULT_BEHAVE_FORMAT = 'progress2'
+DEFAULT_FEATURES_DIRECTORY = 'acceptance_tests/features'
+DEFAULT_PROCESSES = 6  # Higher than 6 causes collex (and possibly other services) to fail intermittently
+
+DEFAULT_TAGS = 'standalone'
+DELIMITER = '_BEHAVE_PARALLEL_BDD_'
+
+
+def is_valid_parallel_environment():
+    if os.getenv('IGNORE_SEQUENTIAL_DATA_SETUP') == None:
+        return False
+
+    is_ignore_sequential_data_setup = strtobool(os.getenv('IGNORE_SEQUENTIAL_DATA_SETUP'))
+
+    return is_ignore_sequential_data_setup
+
+
+def parse_arguments():
+    """
+    Parses commandline arguments
+    :return: Parsed arguments
+    """
+    parser = argparse.ArgumentParser('Run behave in parallel mode for scenarios')
+    parser.add_argument('--command_line_args', '-a', help='Command line arguments', default=DEFAULT_COMMAND_LINE_ARGS)
+    parser.add_argument('--format', '-f', help='Behave format', default=DEFAULT_BEHAVE_FORMAT)
+    parser.add_argument('--acceptance_features_directory', '-d', help='specify directory containing features',
+                        default=DEFAULT_FEATURES_DIRECTORY)
+    parser.add_argument('--processes', '-p', type=int,
+                        help=f'Maximum number of processes. Default [{DEFAULT_PROCESSES}] ', default=DEFAULT_PROCESSES)
+    parser.add_argument('--tags', '-t', help='specify behave tags to run', default=DEFAULT_TAGS)
+    parser.add_argument('--timeout', '-tout', type=int,
+                        help='Maximum seconds to execute each scenario. Default = 300', default=300)
+
+    # TODO BUG: "AND" tags not handled properly
+    args = parser.parse_args()
+
+    return args
+
+
+def is_process_running(process):
+    return process != None and process.is_alive()
```

Comment:
`None` checks are best done with `is None` or `is not None` rather than equality check.

Reply:
fixed